input/LoadBalance/test_balance.incl

```diff
@@ -0,0 +1,315 @@
+         dir = [ "COSMO_CG_%04d", "cycle" ];
+         image_reduce_type = "max";
+         image_size = [ 513, 513 ];
+         image_type = "mesh";
+         axis = "x";
+         name = [ "mesh-%02d.png", "count" ];
+         schedule {
+             step = 20;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     po {
+         dir = [ "COSMO_CG_%04d", "cycle" ];
+         field_list = [ "potential" ];
+         image_size = [ 512, 512 ];
+         image_type = "data";
+         name = [ "po-%02d.png", "count" ];
+         schedule {
+             step = 20;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     check {
+         dir = [ "checkpoint_cosmo_cg-%02d", "count" ];
+         schedule {
+             start = 100;
+             step = 200;
+             var = "cycle";
+         };
+         type = "checkpoint";
+     };
+     dep {
+         dir = [ "COSMO_CG_%04d", "cycle" ];
+         field_list = [ "density_particle" ];
+         image_ghost = false;
+         image_size = [ 512, 512 ];
+         image_type = "data";
+         name = [ "dep-%02d.png", "count" ];
+         schedule {
+             step = 20;
+             var = "cycle";
+         };
+         type = "image";
+     };
+     hdf5 {
+         dir = [ "COSMO_CG_%04d", "cycle" ];
+         field_list = [ "density", "velocity_x", "velocity_y", "velocity_z", "acceleration_x", "acceleration_y", "acceleration_z", "total_energy", "internal_energy", "pressure" ];
+         name = [ "data-%02d-%02d.h5", "count", "proc" ];
+         particle_list = [ "dark" ];
+         schedule {
+             step = 20;
+             var = "cycle";
+         };
+         type = "data";
+     };
+ }
+
+ Particle {
+     dark {
+         attributes = [ "x", "default", "y", "default", "z", "default", "vx", "default", "vy", "default", "vz", "default", "ax", "default", "ay", "default", "az", "default", "is_local", "default"];
+         constants = [ "mass", "default", 0.8666666666667000/(32.0*32.0*32.0) ];
+         position = [ "x", "y", "z" ];
+         velocity = [ "vx", "vy", "vz" ];
+         group_list = ["is_gravitating"];
+	 mass_is_mass = true;
+     };
+     list = [ "dark" ];
+ }
+
+ Physics {
+     cosmology {
+         comoving_box_size = 3.000000000000000;
+         final_redshift = 3.000000000000000;
+         hubble_constant_now = 0.7000000000000000;
+         initial_redshift = 99.00000000000000;
+         max_expansion_rate = 0.01500000000000000;
+         omega_baryon_now = 0.04000000000000000;
+         omega_cdm_now = 0.2600000000000000;
+         omega_lambda_now = 0.7000000000000000;
+         omega_matter_now = 0.3000000000000000;
+     };
+     list = [ "cosmology" ];
```

Comment:
A suggestion to use the new fluid_props physics object (I need to address this in existing parameter files)

```suggestion
     fluid_props {
         dual_energy { type = "bryan95"; }
         eos { gamma = 1.666700000000000; }
     };
     list = [ "cosmology", "fluid_props" ];
```

Reply:
Deferring this to when you update other parameter files so the related updates are all in one place